pkg/envoy/ads/response.go

```diff
@@ -16,71 +20,91 @@ const (
 	ADSUpdateStr = "ADS"
 )
 
-// Wrapper to create and send a discovery response to an envoy server
-func (s *Server) sendTypeResponse(tURI envoy.TypeURI,
-	proxy *envoy.Proxy, server *xds_discovery.AggregatedDiscoveryService_StreamAggregatedResourcesServer,
-	req *xds_discovery.DiscoveryRequest, cfg configurator.Configurator) error {
+// getXDSResource function will invoke the right TypeURI handler for the resources associated with a proxy
+// Returns the list of resources to be sent associated to TypeURI and the proxy
+func (s *Server) getXDSResource(tURI envoy.TypeURI, proxy *envoy.Proxy,
```

Comment:
Would you be so kind to expand abbreviated variable names? I personally find it tough to read `tURI`. Subjectively I believe `typeURI` makes the code easier to read, lower cognitive load.